prqlc/prqlc-parser/src/lib.rs

```diff
@@ -1,35 +1,25 @@
-pub mod err;
-mod expr;
-mod interpolation;
-mod lexer;
-mod stmt;
-#[cfg(test)]
-mod test;
-mod types;
-
-use chumsky::{prelude::*, Stream};
-pub use prqlc_ast as ast;
-use prqlc_ast::span::Span;
-use prqlc_ast::stmt::*;
-use prqlc_ast::token::*;
+use crate::error::Error;
+use crate::lexer::lr::TokenKind;
+use crate::parser::parse_lr_to_pr;
+use crate::parser::pr::Stmt;
 
-use self::err::error::Reason;
-use self::err::error::{Error, WithErrorInfo};
 pub use self::lexer::TokenVec;
-use crate::err::error::ErrorSource;
+
+pub mod error;
+pub mod lexer;
+pub mod parser;
+pub mod span;
+
+mod test;
+// pub use crate::err::*;
```

Comment:
Super minor and let's not delay merging — but IIUC the rust standard is putting `mod`s above `use`s; the PR seems to reverse those.

(another one where I don't have a view apart from doing the standard thing — so if I'm wrong on the standard thing then let's do the correct standard thing :) )

Reply:
I made some changes for these